drawBot/context/tools/variation.py

```diff
@@ -1,19 +1,22 @@
 import CoreText
 from collections import OrderedDict
+from drawBot.misc import memoize
 
 """
 https://developer.apple.com/documentation/coretext/ctfont/font_variation_axis_dictionary_keys?language=objc
 https://developer.apple.com/documentation/coretext/1508650-ctfontdescriptorcreatecopywithva?language=objc
 """
 
 
+@memoize
```

Comment:
I bet this one and the next (`convertVariationTagToInt`) make no significant contribution to the speedup.